src/graphql/error-map.ts

```diff
@@ -335,6 +335,9 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
       message = "Invalid pagination arguments"
       return new ValidationInternalError({ message, logger: baseLogger })
 
+    case "UserWithPhoneAlreadyExistsError":
+      message =
+        "The phone is associated with an existing wallet that has a non zero balance. Sweep the funds and try again."
```

Comment:
Missing return for this `case`

```suggestion
        "The phone is associated with an existing wallet that has a non zero balance. Sweep the funds and try again."
      return new ValidationInternalError({ message, logger: baseLogger })

```

Reply:
Fixed here - e96b93fc9fa1ae4a0609410d4b64b77c7f073e70

Reply:
We generally keep the CustomApolloErrors pretty generic, but get very specific with the domain errors. I'd lean more to `ValidationInternalError` instead of making a new specific apollo error

Reply:
Resolved